greenlight/tests/functional/keyboard_shortcuts/test_browser_window.py

```diff
@@ -39,9 +39,11 @@ def test_search_field(self):
 
         keys = [self.keys.ACCEL]
         if self.marionette.session_capabilities['platformName'] == 'LINUX':
-            keys.append(self.browser.get_localized_entity('searchFocusUnix.commandkey'))
+            keys.append(self.browser.get_localized_entity(
+                        'searchFocusUnix.commandkey'))
```

Comment:
Sounds like a good compromise. So what about lifting it up to 100 chars? We can tweak that easily by specifying the `--max-line-length` option.